cibuildwheel/linux.py

```diff
@@ -174,74 +177,81 @@ def build_on_docker(
             )
             sys.exit(1)
 
-        if build_options.before_build:
-            log.step("Running before_build...")
-            before_build_prepared = prepare_command(
-                build_options.before_build,
-                project=container_project_path,
-                package=container_package_dir,
-            )
-            docker.call(["sh", "-c", before_build_prepared], env=env)
-
-        log.step("Building wheel...")
-
-        temp_dir = PurePath("/tmp/cibuildwheel")
-        built_wheel_dir = temp_dir / "built_wheel"
-        docker.call(["rm", "-rf", built_wheel_dir])
-        docker.call(["mkdir", "-p", built_wheel_dir])
-
-        verbosity_flags = get_build_verbosity_extra_flags(build_options.build_verbosity)
-
-        if build_options.build_frontend == "pip":
-            docker.call(
-                [
-                    "python",
-                    "-m",
-                    "pip",
-                    "wheel",
-                    container_package_dir,
-                    f"--wheel-dir={built_wheel_dir}",
-                    "--no-deps",
-                    *verbosity_flags,
-                ],
-                env=env,
-            )
-        elif build_options.build_frontend == "build":
-            config_setting = " ".join(verbosity_flags)
-            docker.call(
-                [
-                    "python",
-                    "-m",
-                    "build",
-                    container_package_dir,
-                    "--wheel",
-                    f"--outdir={built_wheel_dir}",
-                    f"--config-setting={config_setting}",
-                ],
-                env=env,
-            )
+        abi3_wheel = find_compatible_abi3_wheel(built_wheels, config.identifier)
+        if abi3_wheel:
+            log.step_end()
+            print(f"SKIP building wheel. Using previously built wheel {abi3_wheel.name}")
```

Comment:
I'm thinking there might be a way to rephrase that doesn't have the potential to confuse with CIBW_SKIP here...

```suggestion
            print(f"Found previously built wheel {abi3_wheel.name}, that's compatible with {config.identifier}. Skipping build step...")
```

Reply:
suggestion applied